x-pack/plugins/secops/public/components/page/hosts/hosts_table/index.tsx

```diff
@@ -98,10 +103,42 @@ export const HostsTable = connect(
 
 const getHostsColumns = () => [
   {
-    name: 'Host',
+    name: 'Name',
     truncateText: false,
     hideForMobile: false,
-    render: ({ host }: { host: HostItem }) => <>{defaultTo('--', host.name)}</>,
+    render: ({ host }: { host: HostItem }) => {
+      const hostName = defaultTo('--', host.name);
+      return (
+        <>
+          <DraggableWrapper
+            dataProvider={{
+              and: [],
+              enabled: true,
+              id: `${host._id}`,
+              name: hostName,
+              negated: false,
+              queryMatch: `host.name: "${escapeQueryValue(hostName)}"`,
+              queryDate: `@timestamp >= ${
+                moment(host.firstSeen!).valueOf
```

Comment:
Was this suppose to be `valueOf()`?

Good old javascript, looks like it lets you use functions in compares even by accident

```ts
moment.valueOf > moment.valueOf()
true
```

Reply:
to get the date in ms

Reply:
I think @FrankHassanabad is asking if you meant to apply / invoke the `valueOf` function. (Is it missing `()` i.e. `.valueOf()`?)

Reply:
oh yes!!!, I did not even see it

Reply:
Good catch!!!